src/Uno.UI/Microsoft/UI/Xaml/Controls/MenuBar/MenuBarItem.cs

```diff
@@ -350,6 +364,12 @@ void OnFlyoutOpening(object sender, object args)
 			if (m_menuBar != null)
 			{
 				m_menuBar.IsFlyoutOpen = true;
+
+				var menuBar = m_menuBar;
+				var presenter = m_flyout.m_presenter;
+				PointerEventHandler pointerMovedHandler = (_, e) => menuBar.OnMenuBarItemFlyoutPresenterPointerMoved(e);
+				m_flyout._popup.PopupPanel.PointerMoved += pointerMovedHandler;
+				_menuBarPointerMovedDisposable.Disposable = new DisposableAction(() => presenter.PointerMoved -= pointerMovedHandler);
```

Comment:
This block of code should be marked Uno specific too, with a comment indicating that this is a workaround for the lack of `OverlayInputPassThroughElement`